portal/server/generateApp.ts

```diff
@@ -95,17 +102,21 @@ const generateApp = () => {
   app.use(cookieParser());
 
   app.use(copyCsrfTokenFromQueryToBody());
-  app.use(csrf({
-    cookie: {
-      ...cookie,
-      maxAge: 43200, // 12 hours
-    },
-  }));
+  app.use(
+    csrf({
+      cookie: {
+        ...cookie,
+        maxAge: 43200, // 12 hours
+      },
+    }),
+  );
   app.use(csrfToken());
 
-  app.use(morgan('dev', {
-    skip: (req) => req.url.startsWith('/assets') || req.url.startsWith('/main.js'),
-  }));
+  app.use(
+    morgan('dev', {
+      skip: (req) => req.url.startsWith('/assets') || req.url.startsWith('/main.js'),
```

Comment:
This can be a middleware to achieve abstraction.

Reply:
IMO this won't make the code any clearer since it isn't repeated anywhere (within Portal at least).

I guess this is the kind of set up we'd want to commonise across all services so we've got consistent logging